test/Tryouts/Program.cs

```diff
@@ -13,50 +20,33 @@ static Program()
         XunitLogging.RedirectStreams = false;
     }
 
-    public static void Main(string[] args)
+    private static RavenTestBase.Options GetOptionForSearchEngine(SearchEngineType type)
+    {
+        return new RavenTestBase.Options()
+        {
+            ModifyDatabaseRecord = record =>
+            {
+                record.Settings[RavenConfiguration.GetKey(x => x.Indexing.AutoIndexingEngineType)] = type.ToString();
+                record.Settings[RavenConfiguration.GetKey(x => x.Indexing.StaticIndexingEngineType)] = type.ToString();
+            }
+        };
+    }
+
+    public static async Task Main(string[] args)
```

Comment:
Can you please fix the warning? The best would be to revert the changed made here so we won't get conflicts on merge

Reply:
Reverted